packages/aws-cdk/lib/cli/user-input.ts

```diff
@@ -727,6 +727,13 @@ export interface DeployOptions {
    */
   readonly hotswap?: boolean;
 
+  /**
+   * Number of seconds to wait for a hotswap operation to complete. Normally this applies to resource stabilization.
```

Comment:
Maybe something like this?
```suggestion
   * Number of seconds to wait for a single hotswap resource update to complete. Normally this applies to resource stabilization.
```